tests/src/JIT/Regression/JitBlue/GitHub_21625/GitHub_21625.cs

```diff
@@ -0,0 +1,52 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+
+namespace GitHub_21625
+{
+    public class test
+    {
+        public static Vector128<ushort> CreateScalar(ushort value)
```

Comment:
Even for the minopts case, it doesn't seem this repros the bug every time. It seems the 2 bytes following `value` on the stack are frequently already zeroed.